modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -41,11 +41,11 @@ def submit # rubocop:disable Metrics/MethodLength
           end
           save_auto_claim!(auto_claim)
 
-          if auto_claim.errors.present?
-            raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity.new(
-              detail: auto_claim.errors.messages.to_s
-            )
-          end
+          # if auto_claim.errors.present?
+          #   raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity.new(
+          #     detail: auto_claim.errors.messages.to_s
+          #   )
+          # end
```

Comment:
Was this commented section intended to be left in place?

Reply:
@mchristiansonVA This has been fixed